crates/holochain_types/src/app/app_manifest.rs

```diff
@@ -57,10 +57,9 @@ use super::{InstalledCell, ModifiersMap};
 /// which manifest version to deserialize to.
 #[derive(Clone, Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize, derive_more::From)]
 #[cfg_attr(feature = "fuzzing", derive(arbitrary::Arbitrary))]
-#[serde(tag = "manifest_version")]
+#[serde(tag = "type", content = "value", rename_all = "snake_case")]
```

Comment:
Note that this one will now look like this on the js-side:
```
{
  type: "v1",
  value: [...]
}
```
instead of
```
{
  // Currently "1" is supported
  manifest_version: string;
  
  name: string;
  description?: string;
  roles: Array<AppRoleManifest>;
  membrane_proofs_deferred: boolean;
}
```

Reply:
reverted according to discussion on MM